pom.xml

```diff
@@ -2052,6 +2052,22 @@
           <artifactId>maven-deploy-plugin</artifactId>
           <version>2.8.2</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-dependency-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>default-cli</id>
+              <goals>
+                 <goal>build-classpath</goal>
+              </goals>
+              <configuration>
+                <!-- Only count compile-scoped dependencies when generating classpath -->
```

Comment:
Yep, I think you're right. According to the `maven-dependency-plugin` docs (https://maven.apache.org/plugins/maven-dependency-plugin/list-mojo.html#includeScope):

> Scope to include. An Empty string indicates all scopes (default). The scopes being interpreted are the scopes as Maven sees them, not as specified in the pom. In summary:
> - runtime scope gives runtime and compile dependencies,
> - compile scope gives compile, provided, and system dependencies,
> - test (default) scope gives all dependencies,
> - provided scope just gives provided dependencies,
> - system scope just gives system dependencies.

Based on this language, it seems like we want to be using `runtime` here so that we include both compile and runtime scope dependencies but do not exclude provided ones.